scripts/intfutil

```diff
@@ -6,74 +6,84 @@ import re
 from tabulate import tabulate
 from natsort import natsorted
 
-
-
 # ========================== Common interface-utils logic ==========================
 
-
 PORT_STATUS_TABLE_PREFIX = "PORT_TABLE:"
+PORT_TRANSCEIVER_TABLE_PREFIX = "TRANSCEIVER_INFO|"
 PORT_LANES_STATUS = "lanes"
 PORT_ALIAS = "alias"
 PORT_OPER_STATUS = "oper_status"
 PORT_ADMIN_STATUS = "admin_status"
 PORT_SPEED = "speed"
 PORT_MTU_STATUS = "mtu"
 PORT_DESCRIPTION = "description"
-
+PORT_OPTICS_TYPE = "type"
 
 def db_connect():
     db = swsssdk.SonicV2Connector(host='127.0.0.1')
     if db == None:
         return None
-
     db.connect(db.APPL_DB)
-
     return db
 
 
 def db_keys_get(db, intf_name):
-
     if intf_name == None:
         db_keys = db.keys(db.APPL_DB, "PORT_TABLE:*")
     elif intf_name.startswith('Ethernet'):
         db_keys = db.keys(db.APPL_DB, "PORT_TABLE:%s" % intf_name)
     else:
         return None
-
     return db_keys
 
 
 def db_port_status_get(db, intf_name, status_type):
     """
     Get the port status
     """
-
     full_table_id = PORT_STATUS_TABLE_PREFIX + intf_name
     status = db.get(db.APPL_DB, full_table_id, status_type)
     if status is None:
         return "N/A"
-
     if status_type == PORT_SPEED and status != "N/A":
        status = '{}G'.format(status[:-3])
-
     return status
 
+def db_connect_optics():
```

Comment:
I think this function should be named along the lines of `db_connect_state()` and `db_connect()` above should be renamed `db_connect_appl()` to distinguish the databases which they are each connecting to. This function can be reused for other non-optics-related STATE DB operations.

Reply:
done.